samples: peripheral_hr_coded: central_hr_coded: Use valid configuration (#7856)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

Bluetooth samples
-----------------

* :ref:`peripheral_hr_coded` sample:

  * Added configuration for the nRF5340 target.

>   * Fixed advertising start on the nRF5340 target with the Zephyr LL controller.

Reviewer: I believe this is fine as is, but do you think the fix could be explained a bit more? For example something like:
"Fixed the behavior of the advertising start process on..."
I don't really know whether that would make this any clearer. Up to you to decide.
Same with the next item, if you think this makes any sense.

Author: I added a sentence which explains what was an issue here I hope that it has more sense now

  * Added a workaround for nRF5340 revision 1 Errata 117.

* :ref:`peripheral_hr_coded` sample:

  * Added configuration for the nRF5340 target.
  * Fixed advertising start on the nRF5340 target with the Zephyr LL controller.

>     Previously, the start of advertising was impossible because the :kconfig:option:`CONFIG_BT_EXT_ADV` was disabled for the Zephyr LL controller.

Reviewer: ```suggestion
    Previously, it was not possible to start advertising, because the :kconfig:option:`CONFIG_BT_EXT_ADV` option was disabled for the Zephyr LL controller.
```

Author: Done

* :ref:`bluetooth_central_hr_coded` sample:

  * Added configuration for the nRF5340 target.
  * Fixed scanning start on the nRF5340 target with the Zephyr LL controller.

>     Previously, the start of advertising was impossible because the :kconfig:option:`CONFIG_BT_EXT_ADV` was disabled for the Zephyr LL controller.

Reviewer: ```suggestion
    Previously, it was not possible to start scanning, because the :kconfig:option:`CONFIG_BT_EXT_ADV` option was disabled for the Zephyr LL controller.
```
We are talking about scanning here, right?

Author: Yes I made copy&paste error

Reviewer: Thought so.
